TaskHouseApi/Model/Offer.cs

```diff
@@ -1,11 +1,18 @@
+using System.ComponentModel.DataAnnotations;
+
 namespace TaskHouseApi.Model
 {
     public class Offer : BaseModel
     {
+        [Required]
         public bool Accepted { get; set; }
```

Comment:
this can not be required because when a offer is created it is not know if the offer is accepted or not.

Reply:
i assume that required don't allow null